test/cpp/jit/test_gpu.cpp

```diff
@@ -3121,6 +3103,33 @@ void testGPU_FusionSoftmax() {
   //     "Error of: ",
   //     t2.sub(cg_output).abs().max());
 }
+
+// TODO: Fix the assert any throw below!
```

Comment:
What does the intended fix look like? I tested `ASSERT_ANY_THROW` by uncommenting it, and it worked, so I assume `computeAt` did throw an exception as it is supposed to do.

Reply:
Thanks, forgot to update this test after I finished. Updating now.